### PR TITLE
Update copy to remove dates and references to support portal

### PR DESCRIPTION
**Pull request by nigel-lowry**
### Context

https://trello.com/c/jiKqsrTb

### CODE REVIEW

**Comment by nigel-lowry** [comment]
[Removing mention of the Support Portal and updating licence dates (guidance pages) Part 1 of 2](https://trello.com/c/jiKqsrTb/2399-removing-mention-of-the-support-portal-and-updating-licence-dates-guidance-pages-part-1-of-2)

File: app/views/devices_guidance/device_specification.md [hunk 1]
```diff
@@ -86,7 +89,7 @@ The devices came with any remaining manufacturer's warranty (this was less than
         <td class="govuk-table__cell">HD</td>
         <td class="govuk-table__cell">Minimum 1280 × 800</td>
         <td class="govuk-table__cell">2160 × 1620</td>
-        <td class="govuk-table__cell">HD (1366 x 768) anti-glare</td>
+        <td class="govuk-table__cell">HD (1366 × 768) anti-glare</td>
```
Comment (kenfodder) [inline app/views/devices_guidance/device_specification.md:92]: Either stick with `x` or use `&times;`

**Review by kenfodder — APPROVED** [review]

File: app/views/devices_guidance/device_specification.md [hunk 2]
```diff
@@ -102,7 +105,7 @@ The devices came with any remaining manufacturer's warranty (this was less than
         <td class="govuk-table__cell">Minimum 4GB</td>
         <td class="govuk-table__cell">Minimum 2GB</td>
         <td class="govuk-table__cell">3GB</td>
-        <td class="govuk-table__cell">8GB, 1x8GB, DDR4 Non-ECC</td>
```
Comment (kenfodder) [inline app/views/devices_guidance/device_specification.md:105]: Same here

File: app/views/devices_guidance/device_specification.md [hunk 3]
```diff
@@ -118,7 +121,7 @@ The devices came with any remaining manufacturer's warranty (this was less than
         <td class="govuk-table__cell">Wi-Fi to 802.11ac</td>
         <td class="govuk-table__cell">Wi-Fi to 802.11ac</td>
         <td class="govuk-table__cell">802.11a/b/g/n/ac</td>
-        <td class="govuk-table__cell">Intel Dual Band wireless AC 9560 (802.11ac) 2x2, Bluetooth 5.0</td>
+        <td class="govuk-table__cell">Intel Dual Band wireless AC 9560 (802.11ac) 2×2, Bluetooth 5.0</td>
```
Comment (kenfodder) [inline app/views/devices_guidance/device_specification.md:124]: Same here.

**Review by kenfodder — APPROVED** [review]

**Pull request merged**